src/store/comment.js

```diff
@@ -60,7 +60,15 @@ export default {
 		}
 	},
 	actions: {
-		listComments({ commit }, card) {
+		listComments({ commit, state }, card) {
+			card.limit = 5
+			card.offset = 0
+			if (state.comments[card.id] !== undefined) {
+				if (state.comments[card.id].length === card.limit) {
+					card.offset = card.limit
```

Comment:
wouldn't this only allow fetching the comments once? What if you have >10 comments.